composer.json

```diff
@@ -23,10 +23,9 @@
         "friendsofphp/php-cs-fixer": "^3.4.0",
         "phpbench/phpbench": "^1.2.3",
         "phpunit/phpunit": "^9.5.16",
-        "vimeo/psalm": "^4.21.0",
-        "php-standard-library/psalm-plugin": "^2.0.2",
-        "php-coveralls/php-coveralls": "^2.5.2",
-        "roave/infection-static-analysis-plugin": "^1.23.0"
+        "vimeo/psalm": "^5.0.0",
+        "php-standard-library/psalm-plugin": "dev-psalmv5",
```

Comment:
(Temporary changes to test psalm-plugin and make roave/infection-static-analysis-plugin non-blocking)